### PR TITLE
#167190427 Setup coveralls services

### PR DESCRIPTION
**Pull request by chiboycalix**
#### What does this PR do?
- Setup coveralls services
#### Description of Task to be completed?
- Install coveralls as a dev dependency
- Install nyc as a dev dependency
- Setup coverall script in package.json
- Configure your travis.yml file to use coverall script
- Add coveralls badge to readme.md file
#### How should this be manually tested?
- N/A
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#167190427](https://www.pivotaltracker.com/story/show/167190427)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

**Review by cvjude — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -8,10 +8,8 @@
     "dev": "export DEBUG=dev && nodemon --exec babel-node server/index.js",
     "clean": "rm -rf build && mkdir build",
     "build": "npm run clean && babel -d ./build ./server",
-    "migrate": "node_modules/.bin/sequelize db:migrate",
-    "migrate:undo": "node_modules/.bin/sequelize db:migrate:undo",
-    "generate:model": "node_modules/.bin/sequelize model:generate",
-    "test": "NODE_ENV=test mocha --require @babel/register server/tests --exit"
+    "test": "NODE_ENV=test nyc mocha --require @babel/register server/tests --exit",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls"
```
Comment (chuxmykel) [inline package.json:15]: Hey @chiboycalix Nice work with the coveralls setup. However, I think you overwrote the migration scripts with your new commit. I'd suggest you do a rebase off the develop branch before pushing.

**Review by chuxmykel — CHANGES_REQUESTED** [review]
Take a look at the migration scripts.

File: package.json (hunk 1 above)
Comment (chiboycalix) [reply package.json:15]: Thanks for pointing this out @chuxmykel , It has been fixed

**Review by Hadeneekeh — APPROVED** [review]
LGTM

**Review by chuxmykel — APPROVED** [review]
Great job buddy.

**Review by obayomi96 — APPROVED** [review]

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by encodedBicoding — APPROVED** [review]
LGTM. great job

**Review by topseySuave — APPROVED** [review]

**Pull request merged**